cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -130,7 +130,18 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
      }
       p("When a database has been created, it will show up in the listing provided by the command `SHOW DATABASES`.")
       query("SHOW DATABASES", assertDatabasesShown) {
-        resultTable()
+        limitedResultTable(maybeWantedColumns = Some(List("name", "type", "access", "role", "writer", "currentStatus")))
+      }
+
+      section("Cluster Topology", "administration-databases-create-database-topology", "enterprise-edition") {
+        p("In a cluster environment, it may be desirable to control the number of servers used to host a database. " +
+          "The number of primary and secondary servers can be specified using the following command. " +
```

Comment:
It's better to add the command right after the end of this sentence and move "for more details..." after the query section. Also, please add : after "following command", so it is more demonstrative.